macros-tests/src/test/scala/interpolator/MetadataInterpolatorTest.scala

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Maybe we could move these tests to integration tests.

Reply:
I'll do it as part of #88